### PR TITLE
Updated Existing RHDH Doc with NetworkPolicy param

### PR DESCRIPTION
**Pull request by jenniferubah**
@y-first PTAL

### CODE REVIEW

File: docs/main/existing-rhdh.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ In 1.3, the Orchestrator infrastructure is installed using the Orchestrator Oper
         serviceNamespace: sonataflow-infra
       rhdhOperator:
         enabled: false
+      networkPolicy:
+        rhdhNamespace: custom-rhdh
```
Comment (y-first) [inline docs/main/existing-rhdh.md]: Maybe should be something like:

rhdhNamespace: ${RHDH_NAMESPACE}    # Replace with namespace RHDH is deployed in

File: docs/release-1.2/existing-rhdh.md [hunk 2]
```diff
@@ -54,6 +54,8 @@ In 1.2, the Orchestrator infrastructure is installed using the Orchestrator Oper
         serviceNamespace: sonataflow-infra
       rhdhOperator:
         enabled: false
+      networkPolicy:
+        rhdhNamespace: custom-rhdh
```
Comment (y-first) [inline docs/release-1.2/existing-rhdh.md]: same as above

Comment (jenniferubah) [reply docs/release-1.2/existing-rhdh.md]: updated

**Pull request merged**